templates/mask_generation.ipynb

```diff
@@ -0,0 +1,298 @@
+{
```

Comment:
<p>This is great!!!</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/angelolab/ark-analysis/pull/1052/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='17'/>